gradle.properties

```diff
@@ -1,21 +1,13 @@
-# Project-wide Gradle settings.
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx1536m
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-# AndroidX package structure to make it clearer which packages are bundled with the
-# Android operating system, and which are packaged with your app's APK
-# https://developer.android.com/topic/libraries/support-library/androidx-rn
-android.useAndroidX=true
-# Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
-# Kotlin code style for this project: "official" or "obsolete":
+android.injected.testOnly=false
+#android.lifecycleProcessor.incremental=true
+android.useAndroidX=true
+#kapt.include.compile.classpath=false
+#kapt.incremental.apt=true
+#kapt.use.worker.api=true
+#kapt.verbose=true
 kotlin.code.style=official
+org.gradle.caching=true
+#org.gradle.configureondemand=true
+org.gradle.jvmargs=-Xmx1536m -Dfile.encoding=UTF-8
+org.gradle.parallel=true
```

Comment:
Can you amend all the changes in this file? I'd rather review this change in a separate PR.

Reply:
Revert `gradle.properties` to your version